frontend/src/components/footer.css

```diff
@@ -32,7 +32,7 @@ footer {
     margin: 10px auto;
     line-height: 28px;
     font-size: 14px;
-    color: black;
+    color: ghostwhite;
```

Comment:
eventuell lohnt es sich für deine neuen Farben Variablen zu deklarieren , damit du da immer einfach drauf zugreifen kannst?   z.B. in index.css:     body{--hellgrau: rgba(112, 84, 68, 0.7);}